scripts/systems/population_manager.gd

```diff
@@ -0,0 +1,76 @@
+extends Node
+
+const MIN_POPULATION = 3
```

Comment:
We should move all these constants and the starting values below to a new resource, the read those resources in as a part of initialization

Reply:
Done